CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## moler 1.10.0
+
+### Improved
+* Classes Connection and ObservableConnection are deprecated now.
```

Comment:
add to this description: `Their original code has been moved into AbstractMolerConnection and ThreadedMolerConnection. Change made for code clarity.`

Reply:
Done.